plugins/ltac/extratactics.ml4

```diff
@@ -596,10 +596,16 @@ let inImplicitTactic : glob_tactic_expr option -> obj =
        subst_function = subst_implicit_tactic;
        classify_function = (fun o -> Dispose)}
 
+let warn_deprecated_implicit_tactic =
+  CWarnings.create ~name:"deprecated-implicit-tactic" ~category:"deprecated"
+    (fun () -> strbrk "Implicit tactics are deprecated")
```

Comment:
The current convention (even if fragile) for errors is to add periods. For warnings, this is unregular. Maybe a good time to adopt a convention for warnings.

Reply:
Would there be a way to shortly explain what to use instead (relying on a notation of the style "__ := ltac:(auto)")?